sentry/src/main/java/io/sentry/DefaultSentryClientFactory.java

```diff
@@ -325,6 +328,9 @@ protected Connection createConnection(Dsn dsn) {
         if (protocol.equalsIgnoreCase("http") || protocol.equalsIgnoreCase("https")) {
             logger.debug("Using an {} connection to Sentry.", protocol.toUpperCase());
             connection = createHttpConnection(dsn);
+        } else if (protocol.equalsIgnoreCase("udp")) {
+            logger.debug("Using an {} connection to Sentry.", protocol.toUpperCase());
+            connection = new UdpConnection(dsn, getMaxMessageLength(dsn));
```

Comment:
I think `maxChunkSize` deserves its own option (and default). This is used to trim the toplevel error message string in an event. It's very small and unrelated to the overall JSON payload size or UDP-chunkifying.